src/roiextractors/extraction_tools.py

```diff
@@ -132,15 +132,15 @@ def _validate_video_structure(self) -> None:
     def build_video_shape(self, n_frames: int) -> Tuple[int, int, int, int]:
         video_shape = [None] * 4
         video_shape[self.frame_axis] = n_frames
-        video_shape[self.rows_axis] = self.rows
-        video_shape[self.columns_axis] = self.columns
-        video_shape[self.num_channels_axis] = self.num_channels
+        video_shape[self.rows_axis] = self.num_rows
+        video_shape[self.cols_axis] = self.num_cols
+        video_shape[self.channels_axis] = self.num_channels
```

Comment:
Sorry to keep going back and forth on this kind of thing, but what's your opinion on collapsing `columns` to `cols` everywhere here? I wasn't paying much attention to it before, but I do always think that full words are superior to abbreviations. That said, the mathematician in me likes the symmetry in variable name length between `rows` and `cols`, lol.

Reply:
My own preference was to have full words as well. From the point of view that we spend more time reading code than writing it I think that it pays to be redundant. I thought about that when I saw you change in the other PR but does not feel like a big deal in this case and I did not want to nitpick.

Reply:
I think we might as well go with full-word `columns` then, I agree on that but haven't probably been the best about it even myself.

Reply:
Ok, I tried to do this in the last PR. Let's keep an eye open to see if something got through.